app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabs.kt

```diff
@@ -69,20 +72,54 @@ fun SyncedTabsList(syncedTabs: List<SyncedTabsListItem>, onTabClick: (SyncTab) -
         modifier = Modifier.fillMaxSize(),
         state = listState,
     ) {
-        items(syncedTabs) { syncedTabItem ->
-            when (syncedTabItem) {
-                is SyncedTabsListItem.Device -> SyncedTabsDeviceItem(deviceName = syncedTabItem.displayName)
-                is SyncedTabsListItem.Error -> SyncedTabsErrorItem(
-                    errorText = syncedTabItem.errorText,
-                    errorButton = syncedTabItem.errorButton
-                )
-                is SyncedTabsListItem.NoTabs -> SyncedTabsNoTabsItem()
-                is SyncedTabsListItem.Tab -> {
-                    SyncedTabsTabItem(
-                        tabTitleText = syncedTabItem.displayTitle,
-                        url = syncedTabItem.displayURL,
-                    ) {
-                        onTabClick(syncedTabItem.tab)
+        if (FeatureFlags.taskContinuityFeature) {
+            syncedTabs.forEach { syncedTabItem ->
+                when (syncedTabItem) {
```

Comment:
Are we missing handling for `SyncedTabsNoTabsItem()` here?

Reply:
I am looked at designs, it doesn't seem to indicate that we should remove displaying something when there are no synced tabs.

Reply:
`SyncedTabsNoTabsItem` is just no longer needed in the sticky header revision. See line 93

> if (syncedTabItem.tabs.isNotEmpty()) {
                       ...
                        } else {
                            item { SyncedTabsNoTabsItem() }
                        }

Reply:
Oh missed it.